docusaurus.config.js

```diff
@@ -21,7 +21,7 @@ const config = {
   organizationName: 'uptane', // Usually your GitHub org/user name.
   projectName: 'uptane.github.io', // Usually your repo name.
 
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'ignore',
```

Comment:
We absolutely should not be ignoring broken links.

Reply:
I added this for the temporary purpose, as I plan to add a 'Check Broken Links' workflow once we are done with the initial setup. as I thought that We will address broken links afterward, I also mentioned it in the commit message, but ya addressing them makes senses, I was just ignoring this for now as we might change the path for some folders as well in the future, then files pointing to those folders might get affected.